diff --git a/node_modules/drizzle-kit/bin.cjs b/node_modules/drizzle-kit/bin.cjs
index 4335b1f..1f5db5c 100755
--- a/node_modules/drizzle-kit/bin.cjs
+++ b/node_modules/drizzle-kit/bin.cjs
@@ -11218,7 +11218,7 @@ var require_node2 = __commonJS({
           sourcefile: filename,
           loader: getLoader(filename),
           sourcemap: "both",
-          target: options.target,
+          target: "es6",
           jsxFactory: options.jsxFactory,
           jsxFragment: options.jsxFragment,
           format,
